src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+
+export interface AccordionProps {
+    type: 'base' | 'single'
+    accordionData: AccordionItemProps[]
+}
+
+export interface AccordionItemProps {
+    id: string
+    title: string
+    content: string
+}
+
+const Accordion: React.FC<AccordionProps> = ({
+    type, 
+    accordionData
+}): JSX.Element => {
+    const accordionClass = type === 'single' ? 'accordion__input--single' : ''
+    const accordionType = type === 'single' ? 'radio' : 'checkbox'
+    const accordionId = type === 'single' ? 's' : ''
+    const [ariaState, setAriaState] = useState(true)
+    const toggleHandler=()=>{
+        setAriaState(!ariaState)
+    }
+    return (
+        <>
+            {accordionData.map((accordion, index) => 
+                <div key={index} className="c-accordion">
+                    <input className={`accordion__input ${accordionClass}`} id={`accordion-id-${index}${accordionId}`}  name={`accordion-${accordionType}`} type={accordionType} aria-expanded={!ariaState} onClick={toggleHandler}/>
+                    <label htmlFor={`accordion-id-${index}${accordionId}`} aria-controls={`accordion-aria-control-${index}${accordionId}`} id={`accordion-labelledby-${index}${accordionId}`} role="heading" aria-level={1} aria-hidden={ariaState} dangerouslySetInnerHTML={{ __html: accordion.title }} />
+                    <div id={`accordion-aria-control-${index}${accordionId}`} className="accordion__content" role="region" aria-labelledby={`accordion-labelledby-${index}${accordionId}`} aria-hidden={ariaState} >
+                        <div className="accordion__spacing">
+                            <p dangerouslySetInnerHTML={{ __html: accordion.content }} />
```

Comment:
Single variant is missing the open/close button
<img width="1616" alt="Screen Shot 2022-01-19 at 9 48 42 AM" src="https://user-images.githubusercontent.com/11657599/150154544-1e5a87fa-59dd-4f65-878e-95fafd9904ba.png">

Reply:
Done

Reply:
Well done

Reply:
The button is back but the animation is not working

Reply:
Good job. 
Animation is still glitching.

Reply:
Done, fixed